### DISCUSSION TITLE
Sync Customer Groups

### DISCUSSION THREAD

**Discussion by leonkosterss** [swellstores/community#90] [Ideas]
Hello,

For a new customer we want to use Swell. The only problem is they have Price lists for almost each customer. We want to sync those prices to Swell. The only problem is, that we can't sync all the Customer Groups. Because there are no Customer Groups in Swell yet, we also cannot sync all the specific prices.

Is there a way to sync the Customer Group to a Swell store? Or is coming soon?

**Comment by leonkosterss** [comment]
Maybe because you'll get that much prices per product (like 200), the prices can also be better in a new tab, just like Attributes.

**Comment by jesseswell** [comment]
Hi @leonkosterss 

We do support customer price groups natively. You can access Customer Groups in settings > customers.

<img width="1414" alt="image" src="https://user-images.githubusercontent.com/87753957/213070316-ed1f5bcd-3254-4e5d-8a6e-ec5a331f956d.png">

Once customer groups are added, you can set price rules per product.

<img width="1153" alt="image" src="https://user-images.githubusercontent.com/87753957/213070888-1d4765e7-519b-4962-911b-785e11a313ac.png">

**Comment by leonkosterss** [comment]
Hi @jesseswell ,

Yes I know. And that works fine. 

But I want to create Customer Groups with the Back-end API. Because we have 1000 customer groups (each customer has a different price list).

In that case we can sync the Customer Groups each day, and after the Customer Group sync we can Sync al the prices per Product and per Customer Group.

**Reply by sags95** [reply]
@leonkosterss To create groups with Backend API you can target the `/settings/accounts` endpoint. Here you can either overwrite the `groups` array (which contains all the customer groups) by wrapping the request body with the `$set` parameter, or add a new groups by a PUT request.

**Reply by leonkosterss** [reply]
Aaah nice, thank you! Will try!

**Reply by leonkosterss** [reply]
@sags95 Is there any documentation about this? I cannot find it on the website: https://developers.swell.is/backend-api/introduction

**Reply by sags95** [reply]
@leonkosterss That's a good point, I've mentioned this to the team so that we can get some documentation around this feature!

**Reply by vettloffah** [reply]
The docs are missing a lot of the endpoints but you can generally figure it out by looking in the list of endpoints available in the "console" view in the dashboard. (Just something I learned and am passing along)

![image](https://user-images.githubusercontent.com/3166892/225316750-ddd0d751-74b7-4624-833c-6ea4bd74fcf7.png)

**Reply by ericingram** [reply]
Good point, we need to fix this soon
